Common/NPCs/PullableNPC.cs

```diff
@@ -11,7 +11,7 @@ public class PullableNPC : ModNPC
         public bool Grappled = false;
         protected bool CanBeGrappled = true;
         protected Projectile GrappleProjectile = null;
-        public override string Texture => "Terraria/Projectile_" + ProjectileID.Meteor3;
+        public override string Texture => "Terraria/Images/Projectile_" + ProjectileID.Meteor3;
```

Comment:
Terraria images are now under Terraria/Images. There's a really good chance I've missed a few of these.